docs/Known_SetUp_Issues.adoc

```diff
@@ -0,0 +1,162 @@
+
+If you are a Red Hatter, you can use link:labs.opentlc.com[labs.opentlc.com] to request access to our OpenStack cluster.
+. Log in to link:https://labs.opentlc.com[https://labs.opentlc.com] using your opentlc user.
+. Go to *Services* -> *Catalogs* -> *Novello testing* -> *DEV OSP Sandbox*.
+. Click *Order* -> *Submit*
+
+1. Error when executing test playbook:
++
+[source, shell]
+----
+sassenach:ansible Cibeles$ ansible-playbook main.yml \ 
+    -e @configs/test-empty-config/sample_vars_osp.yml \ 
+    -e @~/.config/openstack/secret.yml
+[...]
+TASK [infra-osp-dry-run : Delete Heat stack that doesn't exist] ********************************************************
+Monday 25 May 2020  15:48:17 +0200 (0:00:00.832)       0:00:11.583 ************
+fatal: [localhost]: FAILED! => {"changed": false, "msg": "openstacksdk is required for this module"}
+	to retry, use: --limit @/Users/Cibeles/repos/agnosticd/ansible/main.retry
+
+PLAY RECAP *************************************************************************************************************
+localhost                  : ok=16   changed=2    unreachable=0    failed=1
+----
+
+You probably forgot to create or activate your virtual env, to fix this simply run:
++
+[source, shell]
+----
+$ source /Users/[USER]/.pyenv/versions/openstack-ansible-2.9/bin/activate
+----
+Please note, now your prompt changes to something like: "(openstack-ansible-2.9) sassenach:ansible Cibeles$"
+
+2. Error still persist, even you have activated virtual env, when executing test playbook:
++
+[source, shell]
+----
+(openstack-ansible-2.9) sassenach:ansible Cibeles$ ansible-playbook main.yml \
+    -e @configs/test-empty-config/sample_vars_osp.yml \
+    -e @~/.config/openstack/secret.yml
+[...]
+TASK [infra-osp-dry-run : Delete Heat stack that doesn't exist] *******************************************************
+Monday 25 May 2020  15:51:21 +0200 (0:00:00.828)       0:00:11.392 ***********
+fatal: [localhost]: FAILED! => {"changed": false, "msg": "openstacksdk is required for this module"}
+----
+
+You need to install openstacksdk. 
++
+[source, shell]
+----
+sassenach:~ Cibeles$ pip install openstacksdk
+----
+
+For further information, please refer to link:openstack doc[https://docs.openstack.org/openstacksdk/latest/user/]
+
+3. After installing openstacksdk, I still get same error when running test playbook.
+
+You may be having version issues. Try the following troubleshooting steps:
++
+[source, shell]
+----
+(openstack-ansible-2.9) sassenach:ansible Cibeles$ /Users/Cibeles/.pyenv/versions/openstack-ansible-2.9/bin/pip list |grep openstack
+DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
```

Comment:
That's a problem. Should really be using Python3 at this point.

Reply:
that's is the only way I got it working, some other errors were found when I tried to use python3, but I could not solve them